js/main.js

```diff
@@ -0,0 +1,59 @@
+const NAMES = ['Олег', 'Алина', 'Ваня', 'Юра', 'Никита', 'Алиса', 'Маша', 'Кирилл', 'Павел',
+  'Вадим', 'Соня', 'Лев', 'Игорь'];
+
+const MESSAGES = ['Всё отлично!', 'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'];
+
+const DESCRIPTIONS = ['Мое фото', 'Удачный кадр', 'мммммммм', '😜', '😈', '😎', '🤡', '🤢', '🤮', '🤬',
+  '☝🏿', 'записывайтесь на ноготочки 💅', '+ 10 社会学分', 'cringe', 'Biergetränk wärmt die Seele',
+  '✝', '🍆💦'];
+
+const NUMBER_OF_GENERATED_PHOTOS = 25;
+
+const getRandomNumber = (min, max) => {
+  const lower = Math.ceil(Math.min(Math.abs(min), Math.abs(max)));
+  const upper = Math.floor(Math.max(Math.abs(min), Math.abs(max)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+
+  return Math.floor(result);
+};
+
+const createRandomIdFromRange = (min, max) => {
+  const previousValues = [];
+
+  return function () {
+    let currentValue = getRandomNumber(min, max);
+    if (previousValues.length >= (max - min + 1)) {
+      return;
+    }
+    while (previousValues.includes(currentValue)) {
+      currentValue = getRandomNumber(min, max);
+    }
+    previousValues.push(currentValue);
+    return currentValue;
+  };
+};
+
+const photoIdGenerator = createRandomIdFromRange(1, 25);
```

Comment:
```suggestion
const createRandomPhotoId = getRandomIdCreator(1, 25);
```

Ну и с остальными ниже аналогично

Reply:
Давай вынесем в константные значения количество айдишников для url и comment? Так мы сможем избежать "магических чисел" которые запрещены по критериям и ситуации, когда в одном месте создали, например, 10 айдишников для комментариев, а в другом месте создаем 100 комментариев. 

```suggestion
const photoIdGenerator = createRandomIdFromRange(1, NUMBER_OF_GENERATED_PHOTO);
const photoUrlGenerator = createRandomIdFromRange(1, NUMBER_OF_GENERATED_URL);
const commentIdGenerator = createRandomIdFromRange(1, NUMBER_OF_GENERATED_COMMENT);
```